spec/spec_helper.rb

```diff
@@ -16,7 +16,7 @@
 ActiveRecord::Migration.check_pending! if defined?(ActiveRecord::Migration)
 
 RSpec.configure do |config|
-  config.include FactoryGirl::Syntax::Methods
+  config.include FactoryBot::Syntax::Methods
```

Comment:
Do we need this twice ? cf: ` spec/support/factory_girl.rb` also should rename the support file.